content/en/apps/guides/data/analytics/setup.md

```diff
@@ -65,4 +65,4 @@ docker compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f d
 To remove all the data volumes, add `-v` at the end of this command.
 
 ## Setup Superset
-To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database.
+To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database. Medic recommends adding a way to track Superset changes via a git repository or any other version control system to make it easier to track changes over time and potentially catch and remediate bugs and regressions. One way to do this is by using [Superset API](https://superset.apache.org/docs/api/) to export and import dashboard and chart definitions. This [guide](https://www.restack.io/docs/superset-knowledge-apache-superset-dashboard-code) provides instructions on how to manage Superset dashboards as code and is a good starting point.
```

Comment:
```suggestion
To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database. 

Tracking Superset changes via a git repository or any other version control system is recommended. This will ensure that change history is captured and potentially catch and remediate bugs and regressions. One way to do this is by using the [Superset API](https://superset.apache.org/docs/api/) to export and import dashboard and chart definitions. [This guide](https://www.restack.io/docs/superset-knowledge-apache-superset-dashboard-code) provides instructions on managing Superset dashboards as code.
```